azurerm/resource_arm_dns_ptr_record_test.go

```diff
@@ -0,0 +1,234 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/arm/dns"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAzureRMDnsPtrRecord_basic(t *testing.T) {
+	ri := acctest.RandInt()
+	config := fmt.Sprintf(testAccAzureRMDnsPtrRecord_basic, ri, ri, ri)
```

Comment:
could we switch this to match the newer resources by making this a function which returns a formatted string?